src/ocean/util/app/DaemonApp.d

```diff
@@ -460,7 +463,8 @@ public abstract class DaemonApp : Application,
         this.registerExtension(this.timer_ext);
 
         // Register stats timer with epoll
-        this.timer_ext.register(&this.statsTimer, this.stats_ext.config.interval);
+        ulong offset = timeToNextInterval(this.stats_ext.config.interval);
```

Comment:
`offset` could be probably named a bit more descriptive, indicating that this is the offset only for the first firing time point.